I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 46, in script
    "Ok here we go!"
  File "renpy/common/00action_other.rpy", line 586, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/playerMovement.rpy", line 3, in movePlayerUp
    if canMove:
NameError: name 'canMove' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 46, in script
    "Ok here we go!"
  File "/Applications/renpy-8.1.3-sdk/renpy/ast.py", line 2560, in execute
    Say.execute(self)
  File "/Applications/renpy-8.1.3-sdk/renpy/ast.py", line 615, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/Applications/renpy-8.1.3-sdk/renpy/exports.py", line 1494, in say
    who(what, *args, **kwargs)
  File "/Applications/renpy-8.1.3-sdk/renpy/character.py", line 1394, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "/Applications/renpy-8.1.3-sdk/renpy/character.py", line 1045, in do_display
    display_say(who,
  File "/Applications/renpy-8.1.3-sdk/renpy/character.py", line 741, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Applications/renpy-8.1.3-sdk/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/core.py", line 2166, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Applications/renpy-8.1.3-sdk/renpy/display/core.py", line 3208, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/screen.py", line 793, in event
    rv = self.child.event(ev, x, y, st)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/behavior.py", line 548, in event
    rv = run(action)
  File "/Applications/renpy-8.1.3-sdk/renpy/display/behavior.py", line 395, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 586, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/playerMovement.rpy", line 3, in movePlayerUp
    if canMove:
NameError: name 'canMove' is not defined

macOS-14.4.1-arm64-arm-64bit arm64
Ren'Py 8.2.3.24061702
prototyping 1.0
Mon Oct 14 23:43:12 2024
